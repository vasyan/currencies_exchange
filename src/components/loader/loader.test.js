import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Loader from './index'

describe('<Error>', () => {
  it('should render', () => {
    expect(shallowToJson(shallow(<Loader />))).toMatchSnapshot()
  })
})
