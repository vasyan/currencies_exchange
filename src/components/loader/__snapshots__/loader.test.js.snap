// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error> should render 1`] = `
<div
  className="wrapper"
>
  <div
    className="loader"
  />
</div>
`;
